test/SlowTests/Server/Documents/PeriodicBackup/ServerWideBackup.cs

```diff
@@ -986,60 +987,48 @@ public async Task CanStoreAndEditServerWideBackupForIdleDatabase()
             {
```

Comment:
Waiting for idle is very slow while we also take into account the size of the database (settings + base on size value calculated).
So I modified the test to test all scenarios instead of creating a new one.